serverless-functions/Makefile

```diff
@@ -0,0 +1,57 @@
+ROOT_DIR	:= $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+LAMBDA_PACKAGE_CMD := docker run --rm -v ${PWD}:/var/task:delegated lambci/lambda:build-python3.6
+VENV_COMMAND := virtualenv venv -p python3
+VENV_ACTIVATE := source venv/bin/activate
+CIS_MODULE_PATH := python-modules
+PY_DIR := build/python/lib/python3.6/site-packages
+PIP_CMD := pip install --upgrade
+LAYER_NAME := cis_development
+
+.PHONY:all
+all:
+	@echo 'Available make targets:'
+	@grep '^[^#[:space:]].*:' Makefile
+
+.PHONY:clean-layer
+clean-layer:
+	rm -rf layer/*
+	rm -rf build/*
+	rm cis.zip
+	rm serverless-functions/cis.zip
+
+.PHONY:package-layer
+package-layer:
+	mkdir -p $(PY_DIR)
```

Comment:
probably want to add PY_DIR to the clean target, as well as have the clean target call clean-layer (so clean cleans all)

note: you can `-rm xx` or `rm -f` to ensure make wont choke if the files dont exist, this is handy for clean since you just want it to keep running even if half of the things were already cleaned